public/register/register.pug

```diff
@@ -8,9 +8,9 @@
         .col-sm-9
           input.form-control(type="text", name="username", ng-model="user.username", placeholder="User name", required)
       .form-group
-        label.col-sm-3.control-label Full name
+        label.col-sm-3.control-label Traveler names
```

Comment:
Why "Traveler names" but not "Traveler name"?

Reply:
Just submitted a new commit -- I actually think this "Full name" actually refers to a user registering for the website, so it's been restored